torch/export/_trace.py

```diff
@@ -630,10 +630,10 @@ def _compiling_state_context():
     # Overwrite output specs afterwards.
     from torch._dynamo import config as _dynamo_config
     from torch._functorch._aot_autograd.input_output_analysis import _graph_output_names
-    from torch._guards import detect_fake_mode
 
     flat_fake_args = pytree.tree_leaves((fake_args, fake_kwargs))
-    fake_mode = detect_fake_mode(flat_fake_args)
+    fake_mode = torch.export._trace._detect_fake_mode_from_gm(gm)
```

Comment:
hmm why export._trace?

Reply:
oops, i can change it to utils._detect_fake_mode_from_gm